# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bcron/make_install.patch
# Copyright (C) 2006 - 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2006-10-16 17:34:11.000000000 +0300
+++ ./Makefile	2006-10-16 17:53:32.000000000 +0300
@@ -3,6 +3,7 @@
 # Generated by spac see http://untroubled.org/spac/
 
 SHELL=/bin/sh
+BINDIR=/usr/local/bin
 
 DEFAULT: all
 
@@ -89,8 +90,13 @@
 docs: bcron.info bcron.html
 
 install: installer instcheck
-	./installer
-	./instcheck
+	install -m 0755 bcron-exec $(BINDIR)/
+	install -m 0755 bcron-sched $(BINDIR)/
+	install -m 0755 bcron-spool $(BINDIR)/
+	install -m 0755 bcron-start $(BINDIR)/
+	install -m 0755 bcron-update $(BINDIR)/
+	install -m 0755 bcrontab $(BINDIR)/
+	ln -snf bcrontab $(BINDIR)/crontab
 
 installer: load insthier.o
 	./load insthier -lbg-installer -lbg
